### DISCUSSION TITLE
请问一下径向充磁的磁铁是多大的呢

### DISCUSSION THREAD

**Discussion by zhusunzsw** [peng-zhihui/HelloWord-Keyboard#31] [Q&A]
电机就是2204的不知道磁铁该买多大的

**Comment by mzIndustry** [comment]
"径向充磁的磁铁" 像一个甜甜圈一样，中心对齐径向安装（粘接？）在电机底部。“甜甜圈” 的内外径需要和你买的电机转子匹配，内径略大于转子中轴外径（方便卡进去安装），"甜甜圈“ 外径则比内径大个5、6mm即可。
